addons/mrp/views/res_config_settings_views.xml

```diff
@@ -33,6 +33,9 @@
                             <setting help="Subcontract the production of some products" documentation="/applications/inventory_and_mrp/manufacturing/management/subcontracting.html">
                                 <field name="module_mrp_subcontracting"/>
                             </setting>
+                            <setting help="Process manufacturing orders from the barcode application">
+                                <field name="module_stock_barcode_mrp" />
```

Comment:
The barcode nomenclature field is missing

Reply:
its added in the `stock_barcode_mrp` now